rfc/20/README.md

```diff
@@ -0,0 +1,148 @@
+---
+domain: github.com
+shortname: 20/MINIPROV
+name: Mini-Provisioning API Support
+status: draft
+editor: Konstantin Nekrasov <konstantin.nekrasov@seagate.com>
+---
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Abstract
+
+Hare component must implement Mini-Provisioner API (see the [specification document](https://seagatetechnology.sharepoint.com/:w:/r/sites/gteamdrv1/tdrive1224/_layouts/15/guestaccess.aspx?email=nitin.nimran%40seagate.com&e=4%3ADCGSG0&at=9&CID=C40F4AE6-CD62-43A6-8B09-A76C04A292F1&wdLOR=cA34642E5-B304-413A-BED3-2FD178DACFDF&share=Ea2OBEFIPmNBucxpoE_n5nsB126Vr3sA_-KY-4_hHYkNVA) for more details). In a nutshell Hare must have scripts that setup a singlenode cluster using [ConfStore](https://seagatetechnology.sharepoint.com/:w:/r/sites/gteamdrv1/tdrive1224/_layouts/15/Doc.aspx?sourcedoc=%7BA68B2DB0-B041-4136-8F51-94F0A4685D1E%7D&file=ConfStore_Specification.docx&action=default&mobileredirect=true) as a source of configuration parameters and publish those scripts via a `setup.yaml` file of a predefined format.
```

Comment:
specification document : This link took me to the page to enter credentials for nitin.nimran@seagate.com

![image](https://user-images.githubusercontent.com/62941569/105280968-1693a780-5bd1-11eb-9f8c-e444af6a9ad9.png)

Reply:
Hm, I don't think I can fix it by myself.

@nitin-seagate  can you please give permissions to CORTX.Hare team to your document? It seems like not all team members have the access..